utils/generatePairs.ts

```diff
@@ -1,34 +1,51 @@
 import { App } from "@slack/bolt";
 import { UsersDataManager } from "../database/usersDataManager";
+import { PoolDataManager } from "../database/poolDataManager";
 
 //TODO: change this to actual channel id when deploying
 const TESTING_CHANNEL_ID = "C02J6R0SUSX";
 export const BOT_USER_ID = "U02J904RH1S";
 const userManager = new UsersDataManager();
+const poolManager = new PoolDataManager();
 
 export const generatePairs = async (app: App) => {
```

Comment:
I haven't tested this particular functionality yet, but will soon and the `poolDataManger` is already tested.